alfajores.env

```diff
@@ -43,6 +43,11 @@ IS_CUSTOM_CHAIN=true
 # The path can be absolute, or relative to the docker-compose.yml file.
 DATADIR_PATH=./envs/alfajores/datadir
 
+# Specifies the endpoint of the eigenda proxy to use.
+EIGENDA_PROXY_ENDPOINT=https://eigenda-proxy.alfajores.celo-testnet.org
+# If set to 1 then a local eigenda proxy will be run and the value of EIGENDA_PROXY_ENDPOINT ignored.
+USE_LOCAL_EIGENDA_PROXY=
+
```

Comment:
We could say that we use a local EigenDA-proxy if `EIGENDA_PROXY_ENDPOINT` is unset. This avoids having a setting that gets ignore by another one.

Reply:
That was what I initially tried to achieve, but I couldn't see a way to set the replicas to 0 if `EIGENDA_PROXY_ENDPOINT` was set and otherwise set replicas to 1. I needed an env var with value either 1 or 0 and then I could use conditional expansion to provide the other value.

https://github.com/celo-org/simple-celo-node/pull/9/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R47

Reply:
Let me know if you see a way to achieve that with just `EIGENDA_PROXY_ENDPOINT`

Reply:
Ok I actually managed to achieve this once I realised that the env var could reference values within it.